### ISSUE TITLE
DOE User projects may not always have an award DOI

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/nmdc-server#1320]
Recently had a user submit metadata to NMDC via the submission portal.
This was for a retrospective / closed project.
This project was awarded BEFORE JGI started minting award DOIs for their data

Problem:
- User says YES, data already generated, YES I used a DOE user facility
  - But does NOT have an award DOI
- Award DOI is required

Solution1
Make award DOI optional

Solution2
Add a "awarded without an award DOI" check box

If solution 2, when you get to "multiomics" the JGI and EMSL data types should still be available.

FYI @pkalita-lbl @naglepuff 
Please move to whatever repo this should be in. Probably server?

**Comment by mslarae13** [comment]
EcoFAb. I thought JGI did NOT retroactively assign DOIs?

**Comment by aclum** [comment]
You are correct, there are some proposals in the database that do not have an award DOI or there are other circumstances result in this info being missing.
